modules/media/php/media.class.inc

```diff
@@ -103,6 +103,13 @@ class Media extends \NDB_Menu_Filter
             $fileTypeList[$filetype['file_type']] = $filetype['file_type'];
         }
 
+        //Language selector
+        $languages    = \Utility::getLanguageList();
+        $languageList = [];
```

Comment:
shouldn't this be `array();` ?
Same thing with line `221`.

Reply:
Noooooooooooooooooooooooooooooooooooo!